1/contrib/jenkins/kube-slave-common.sh

```diff
@@ -91,27 +91,28 @@ function generate_kubernetes_config() {
       <name>openshift</name>
       <templates>
         <org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
-          <name>agent</name>
-          <image>jenkinsci/jnlp-slave</image>
+          <name>maven</name>
+          <image>openshift/jenkins-slave-maven-centos7</image>
           <privileged>false</privileged>
           <command></command>
           <args></args>
+          <remoteFs>/home/jenkins</remoteFs>
           <instanceCap>2147483647</instanceCap>
-          <label>agent</label>
+          <label>maven</label>
           <volumes/>
           <envVars/>
           <nodeSelector/>
           <remoteFs>/tmp</remoteFs>
         </org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
         <org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
-          <name>redhatdistortion-maven</name>
-          <image>docker.io/redhatdistortion/maven-jenkins-slave</image>
+          <name>nodejs</name>
```

Comment:
having a nodejs template simply a function of the fact that we are providing a nodejs sample, correct?